src/angularComponent/ngcOmniboxSuggestionsDirective.js

```diff
@@ -1,14 +1,53 @@
+import angular from 'angular';
 import NgcOmniboxSuggestionsController from './ngcOmniboxSuggestionsController.js';
 
-export default function ngcOmniboxSuggestionsDirective() {
+ngcOmniboxSuggestionsDirective.$inject = ['$templateCache'];
+export default function ngcOmniboxSuggestionsDirective($templateCache) {
   return {
     restrict: 'E',
     require: '^^ngcOmnibox',
     scope: true,
     controller: NgcOmniboxSuggestionsController,
     controllerAs: 'suggestions',
     compile(tElement) {
-      tElement.attr('role', 'listbox');
+      const el = tElement[0];
+      el.setAttribute('role', 'listbox');
+
+      const categoryEl = el.querySelector(
+        'ngc-omnibox-suggestion-category, [ngc-omnibox-suggestion-category]'
+      );
+      const itemEl = el.querySelector(
+        'ngc-omnibox-suggestion-item, [ngc-omnibox-suggestion-item]'
+      );
```

Comment:
hm, should we throw an error if we detect more than one `ngc-omnibox-suggestion-category` or `item`?